### ISSUE TITLE
SelectMultiple -- Focus indicator should appear around options when navigating with keyboard

### ISSUE THREAD

**Issue by taysea** [grommet/grommet#7531]
There should be ability for focus indicator to appear around select multiple options instead of just the "active"/"hover" styling that currently appears. It isn't ensured that "active"/"hover" styling meets the necessary accessibility affordances.

I would expect the focus indicator to be placed around the entirety of the option or the checkbox inside each option.

grommet theme

<img width="379" alt="Image" src="https://github.com/user-attachments/assets/ccde0e0e-11f9-4eaa-8b05-72d691d911ea" />

grommet-theme-hpe

<img width="296" alt="Image" src="https://github.com/user-attachments/assets/c2cddd58-8941-4e10-bf75-56133446cb9b" />

**Comment by omp28** [comment]
@taysea The changes are ready for your review.
